### PR TITLE
Coloured holobarriers

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

If the projector is spray painted, the barriers will match the colour of its projector

image

## Why It's Good For The Game

- Easier to keep track which barriers belong to which projector, which is useful when you have like 4 running

## Changelog

:cl: LT3
qol: Holobarriers will match the spray paint colour of their projector
/:cl:

### CODE REVIEW

**Comment by mc-oofert** [comment]
who would reasonably ever in a round spraypaint their holoprojector

this probably should be a direct option on the item instead

**Comment by lessthnthree** [comment]
> who would reasonably ever in a round spraypaint their holoprojector
> 
> this probably should be a direct option on the item instead

yeah I drafted it, going to do it properly on the sign object itself

**Comment by ju45he** [comment]
The assistant with a bluespess batong vs the cat girl with 27 rainbow colored holobarriers quickly creating more, who would win? This is OBVIOUSLY a cat girl powercreep which needs to be dealt with by a speed merge.

File: code/game/objects/items/holosign_creator.dm [hunk 1]
```diff
@@ -64,23 +67,33 @@
...
 	return .
 
 /obj/item/holosign_creator/attack(mob/living/carbon/human/M, mob/user)
 	return
 
 /obj/item/holosign_creator/attack_self(mob/user)
 	if(LAZYLEN(signs))
-		for(var/H in signs)
-			qdel(H)
+		for(var/obj/structure/holosign/hologram as anything in signs)
```
Comment (JohnFulpWillard) [inline code/game/objects/items/holosign_creator.dm:79]: Shouldn't these be using ``as anything`` since we know what the lists should have?

Comment (lessthnthree) [reply code/game/objects/items/holosign_creator.dm:79]: yes, I changed them to anything. though I missed a couple there.

**Pull request merged**